recipe/meta.yaml

```diff
@@ -10,10 +10,13 @@ source:
   fn: {{ name }}-{{ version }}.zip
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
   sha256: {{ sha256 }}
+  # Patches are applied in bld.bat to handle whitespace issues
+  #patches:
+  #- 0001-import-from-collections.abc.patch
```

Comment:
We should fix this. Can you try running this script on that patch?

https://gist.github.com/ocefpaf/162259d7110e055ea2aff106b174a19a